support/donations.md

```diff
@@ -0,0 +1,31 @@
+---
+breadcrumb: Donations
+---
+# Sponsorship and Donations
+
+The OpenSSL project relies on funding to maintain and improve OpenSSL.
+
+You can support the OpenSSL project financially with the purchase of a
+[support contract](contracts.html), or by a sponsorship donation.
+
+We can accept smaller sponsorship donations via
+[GitHub Sponsors](https://github.com/sponsors/openssl).
+
+We do not have a PayPal account. Please do not donate to any PayPal
+account claiming to be associated with us!
+
+Please note that the [OpenSSL Software Foundation](/community/contacts.html)
+(OSF) is incorporated in the the state of Delaware, United States, as a
+non-profit corporation. It does not qualify as a tax-exempt charitable
+organisation under Section 501(c)(3) of the U.S. Internal Revenue Code.
+
+We provide [Acknowledgements for sponsors](acks.html) depending on the level
+of funding:
+
+| Level                         | Acknowledgement                                                   |
+|-------------------------------|-------------------------------------------------------------------|
```

Comment:
This line should be dropped I think to fix the visual glitch at beta.openssl.org for this table.

Reply:
Huh!  It shouldn't render that extra line

Reply:
Oh, this is supposed to be a header-less table...  our CSS adds the bottom-line on tr:first-child, but because the table header is separated from the table body (with \<thead> and \<tbody>), the tr count gets confused.

[sigh] we have too much absolute junk in our CSS 😕

Reply:
Fixed, but looks a bit odd.  I'll comment on that